src/app/page/MakeMission/MissionForm.js

```diff
@@ -121,37 +169,38 @@ function MissionForm({ handleChange, values, onSubmit, getFile /*, assignHelper,
               type: "city",
               // Other options from https://community.algolia.com/places/documentation.html#options
             }}
-            onChange={(query) => setPickUp({ ...pickUp, location: query })}
-            onCursorChanged={({ rawAnswer, query, suggestion, suggestonIndex }) =>
-              console.log("Fired when arrows keys are used to navigate suggestions.")
-            }
-            onClear={() => console.log("Fired when the input is cleared.")}
-            onLimit={({ message }) =>
-              console.log("!!Algolia Places reached current rate limit.".toUpperCase())
-            }
-            onError={({ message }) =>
-              console.log(
-                "Fired when we could not make the request to Algolia Places servers for any reason but reaching your rate limit."
-              )
+            onChange={(query) =>
```

Comment:
Any reason why we are not using handleLocationChange() here?

Reply:
thanks